packages/database/src/modelClasses/Option.js

```diff
@@ -20,20 +20,6 @@ export class OptionRecord extends DatabaseRecord {
           return error.message;
         }
       },
-      async (value, model) => {
```

Comment:
Import options sets has seperate standalone validation to check for duplicates. I also manually tested and import validation still works.